Functions.Rmd

```diff
@@ -367,9 +367,9 @@ But using the same name for two different things will make for confusing code, a
 
 ### A fresh start {#fresh-start}
 
-What happens to the values in between invocations of a function? What will happen the first time you run this function? What will happen the second time?[^answer4] (If you haven't seen `exists()` before: it returns `TRUE` if there's a variable of that name, otherwise it returns `FALSE`.)
+What happens to values between invocations of a function? What happens the first time you run a function? What happens the second time?[^answer4] (If you haven't seen `exists()` before: it returns `TRUE` if there's a variable with that name, otherwise it returns `FALSE`.)
```

Comment:
The second and third questions are supposed to refer specifically to the example below